src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobResumeBuild.java

```diff
@@ -0,0 +1,45 @@
+package com.tikal.jenkins.plugins.multijob;
+
+import hudson.model.AbstractBuild;
+import hudson.model.AbstractProject;
+import hudson.model.Cause;
+import hudson.model.ProminentProjectAction;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+
+import java.io.IOException;
+
+public class MultiJobResumeBuild implements ProminentProjectAction {
+
+	private final AbstractBuild<?, ?> build;
+
+	public MultiJobResumeBuild(AbstractBuild<?, ?> build) {
+		this.build = build;
+	}
+
+	public String getIconFileName() {
+		return "plugin/jenkins-multijob-plugin/tool32.png";
+	}
+
+	public String getDisplayName() {
+		return "Resume build";
```

Comment:
NIT: It's better to keep UI strings in <code>Messages.properties</code> in order to support the localization